### ISSUE TITLE
'assert' was not declared in this scope

### ISSUE THREAD

**Issue by rlittman16** [statgen/demuxlet#64]
Hello,

I had an error installing demuxlet. During make, I get the following error:

hts_utils.cpp:274:13: error: 'assert' was not declared in this scope
             assert(stop);
             ^~~~~~
hts_utils.cpp:274:13: note: suggested alternative: '__sqrt'
             assert(stop);
             ^~~~~~
             __sqrt
make[1]: *** [Makefile:367: hts_utils.o] Error 1
make[1]: Leaving directory '/u/project/xyang123/xyang123-NOBACKUP/rlittman/demuxlet/demuxlet'
make: *** [Makefile:239: all] Error 2

Can this just be solved by adding #include <cassert> into hts_utils.cpp?

Thanks!

**Comment by endeneon** [comment]
Yes, I have the same problem. I tried to add #include but to no avail (it will later cause conflict in filter.cpp). I am compiling on an RHEL7 with GCC 9.2.0.

**Comment by arogozhnikov** [comment]
same problem, ubuntu 18, gcc 7

**Comment by prashanthsama1** [comment]
Did any of you found a solution for this. I am facing the same problem on my server.
Thanks in advance..!!!

**Comment by welchr** [comment]
Try putting the following: 

```c++
#include <cassert>
```

Into the files `filter.h` and `hts_utils.h`.

**Comment by Giov12** [comment]
Hello. I am also getting a similar error but with `VCF_OVERLAP` . My error is with the `filter.cpp` file though.

`filter.cpp:3149:86: error: 'VCF_OVERLAP' was not declared in this scope`

Any advice would be appreciated. Thank you.
